packages/tracing/test/span.test.ts

```diff
@@ -390,4 +392,57 @@ describe('Span', () => {
       expect(span.data).toStrictEqual({ data0: 'foo', data1: 'bar' });
     });
   });
+
+  describe('getBaggage and _getBaggageWithSentryValues', () => {
+    beforeEach(() => {
+      hub.getClient()!.getOptions = () => {
```

Comment:
Do you think it's necessary to add a cleanup for this? Wanna avoid flakeyness in the future.